### PR TITLE
Tc09_User can't change password when New Password and Confirm Password are different

### PR DESCRIPTION
**Pull request by taile47711**

### CODE REVIEW

File: src/test/java/tests/ChangePasswordTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        loginPage.login(Constant.USERNAME, Constant.PASSWORD);
+
+        LogHelper.info("Click change password tab");
+        changePasswordPage.clickChangePasswordTab();
+
+        LogHelper.info("Enter valid value into all fields");
+        changePasswordPage.changePassword(Constant.PASSWORD, "a123:\"/{}!@$\\", "b456:\"/{}!@$\\");
+
+        LogHelper.info("Get error message");
+        String actual = changePasswordPage.getErrorMessage();
```
Comment (tanmle) [inline src/test/java/tests/ChangePasswordTest.java]: var name should be more specific.

Comment (taile47711) [reply src/test/java/tests/ChangePasswordTest.java:29]: Enter valid information into "Current Password" textbox but enter "a123:"/{}!@$\" into "New Password" textbox and "b456:"/{}!@$\" into "Confirm Password" textbox. I took it from the test case so should I need to change it?

Comment (taile47711) [reply src/test/java/tests/ChangePasswordTest.java]: Fixed. Please check again

Comment (tanmle) [reply src/test/java/tests/ChangePasswordTest.java]: what did you fix?

Comment (taile47711) [reply src/test/java/tests/ChangePasswordTest.java]: I fixed it. Please check again

File: src/main/java/page_object/ChangePasswordPage.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package page_object;
+
+import helper.DriverHelper;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class ChangePasswordPage extends BasePage {
+    //locator
+    private final By txtCurrentPassword = By.id("currentPassword");
+    private final By txtNewPassowrd = By.id("newPassword");
```
Comment (MinhQuan119) [inline src/main/java/page_object/ChangePasswordPage.java]: typo

File: src/main/java/page_object/ChangePasswordPage.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    private final By txtConfirmPassword = By.id("confirmPassword");
+    private final By btnChangePassowrd = By.cssSelector("input[type='submit']");
```
Comment (MinhQuan119) [inline src/main/java/page_object/ChangePasswordPage.java]: typo

File: src/main/java/page_object/ChangePasswordPage.java (hunk 2 above)
Comment (taile47711) [reply src/main/java/page_object/ChangePasswordPage.java]: fixed

File: src/main/java/page_object/ChangePasswordPage.java (hunk 3 above)
Comment (taile47711) [reply src/main/java/page_object/ChangePasswordPage.java]: fixed

File: src/test/java/tests/ChangePasswordTest.java (hunk 1 above)
Comment (MinhQuan119) [reply src/test/java/tests/ChangePasswordTest.java]: I saw that you changed on `"Change password with new password and confirm password are different"` step. It's not about that.
He told you at `"Get error message"` step.
The variables name are  just `actual` and `expected`.
`actual` and `expected` of what, you need to specify them

**Review by MinhQuan119 — APPROVED** [review]

File: src/test/java/tests/ChangePasswordTest.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        LogHelper.info("Click change password tab");
+        changePasswordPage.clickChangePasswordTab();
+
+        LogHelper.info("Change password with new password and confirm password are different");
+        String newPassword = "a123:\"/{}!@$\\";
+        String confirmPassword = "b456:\"/{}!@$\\";
+        changePasswordPage.changePassword(Constant.PASSWORD, newPassword, confirmPassword);
+
+        LogHelper.info("Get error message after click Change Password button in change password page");
+        String actualMessage = changePasswordPage.getErrorMessage();
```
Comment (tanmle) [inline src/test/java/tests/ChangePasswordTest.java]: actualErrorMessage

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/tests/ChangePasswordTest.java (hunk 4 above)
Comment (taile47711) [reply src/test/java/tests/ChangePasswordTest.java]: fixed

**Review by tanmle — APPROVED** [review]

**Pull request merged**